DavidGonzalezZumaya/Day3/ChallengeDay3/Circle.java

```diff
@@ -0,0 +1,32 @@
+public class Circle extends Figure {
```

Comment:
The default unnamed package should not be used.
Try to arrange your classes into packages like:
com.bootcamp.dayone.excerciseone.....
or
com.inheritance.one.... two... etc